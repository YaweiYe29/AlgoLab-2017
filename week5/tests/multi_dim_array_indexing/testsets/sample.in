1
2 5 3
1 1 76
1 3 34
2 1 45
0 4 33
0 1 100
2 1
0 0
1 3
