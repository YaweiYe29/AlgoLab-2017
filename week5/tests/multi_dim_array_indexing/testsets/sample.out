45
-1
34
